tech.json

```diff
@@ -536,6 +536,13 @@
                             "canNonTrivialIceClip"
                           ],
                           "note": "Using a frozen Multiviola to clip through a one-tile-high wall."
+                        },
+                        {
+                          "name": "canMellaIceClip",
+                          "requires": [
+                            "canNonTrivialIceClip"
```

Comment:
canManipulateMellas should be here as it is described.  But you may be able to dumbly position them ok.

Reply:
I don't think canManipulateMellas belongs here. That is a method for manipulating them so their peak height increases over time - used to get to the item without the ice clip or to the left door of this same room. The vertical positioning you need to freeze the Mella at is within the range they move initially. Without it is also more in line with the other ice clip techs.

We found out about this trick through sapphronxd and BuggMann: They couldn't figure out how to manipulate the Mellas, so they instead tried to see if the ice clip was possible.

Reply:
That's right, the original clip kinda settles this then.